tracker/internal/services/v1alpha/service_search.go

```diff
@@ -299,7 +299,7 @@ func (s *searchService) DepthFirstSearch(server tracker.SearchService_DepthFirst
 
 			return status.Error(codes.InvalidArgument, "unexpected request body")
 		default:
-			stack = next
+			stack = append(stack, next...)
```

Comment:
Realized there was a bug in the DFS impl. We were replacing the stack rather than "pushing" items to it